com.civicrm.hrjobroles/tests/phpunit/api/v3/HrJobRolesTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+require_once 'HrJobRolesTestBase.php';
+
+class api_v3_HrJobRolesTest extends HrJobRolesTestBase {
```

Comment:
I'm not sure if we really need these tests and if they're adding any value. The HrJobRoles API is just using the BAO methods, so the BAO is the one that needs to be tested